src/handle-data.ts

```diff
@@ -101,11 +105,16 @@ export function getColumnInfo(layout: TableLayout, colIdx: number, pageColIdx: n
       columnWidth,
       id: `col-${pageColIdx}`,
       label: qFallbackTitle,
-      align: !textAlign || textAlign.auto ? autoAlign : textAlign.align,
       stylingIDs: qAttrExprInfo.map((expr) => expr.id),
       // making sure that qSortIndicator is either A or D
       sortDirection: qSortIndicator && qSortIndicator !== 'N' ? qSortIndicator : 'A',
       totalInfo: getTotalInfo(layout, colIdx, pageColIdx, numDims),
+      autoHeadCellTextAlign,
+      autoTotalsCellTextAlgin,
+      textAlign: {
+        auto: !textAlign || textAlign.auto,
+        align: textAlign?.align,
+      },
```

Comment:
I don't really see why we need all of this information for later. Wouldn't it be possible to figure out what the final align is for all three cases (head, totals, body) and store those three straight on the column

Reply:
we need data cell level. I do not see the easy solution to store them on the column